dev/build.sh

```diff
@@ -39,6 +39,10 @@ fi
 set -e
 
 pushd $ROOT > /dev/null
+
+# Remove the old v3 binary to avoid confusion.
+if [[ -f "v3" ]]; then rm v3; fi
+
```

Comment:
Since you will be building fresh binaries below why don't you delete all previous binaries like below:
```shell
# deletes all previous executable binary versions.
rm v[0-9]*
```

Reply:
ah, great suggestion, thanks